package.json

```diff
@@ -15,5 +15,8 @@
   "homepage": "https://tc39.github.io/ecma262/",
   "dependencies": {
     "ecmarkup": "^3.0.0"
+  },
+  "devDependencies": {
+    "@alrra/travis-scripts": "^1.2.2"
```

Comment:
Additionally, I would lock down the version of this dependency (and probably npm shrinkwrap) to protect against malicious updates.